docs/block-api.md

```diff
@@ -52,22 +52,34 @@ description: 'Block showing a Book card.'
 
 #### Category
 
-* **Type:** `String [ common | formatting | layout | widgets | embed ]`
```

Comment:
Why remove the accepted values here?

Reply:
Doesn't look while reading :see_no_evil: also felt repetitive as we are listing it

Reply:
Do we do it elsewhere? Normally I'd go with prettiness but this is an API doc so including those values might be good...

Reply:
Have added back values for now, just saw icon property accepts `string` or `svg` which we have mentioned as text.

I guess will give another round of reading to docs to find inconsistencies